### DISCUSSION TITLE
ESP3d port to laser cutter?

### DISCUSSION THREAD

**Discussion by nionio6915** [luc-github/ESP3D#700] [Ideas]
luc and team-

is there a way to port ESP3d to work with laser cutters and grbl? 

has such a thing been done yet? 

the reason for my query is the board in my dioe laser is a MKS-DLC and has the AUX-1 connector with TX-RX on it. 

the accomanying TFT has custom interface for laser.

**Comment by luc-github** [comment]
grbl is already supported in webui 2.1 but ui is pretty generic, in 3.0 it will be more specific
